server/specs/server.test.js

```diff
@@ -0,0 +1,20 @@
+const request = require('request');
+
+const serverUrl = 'http://127.0.0.1:24623';
+
+
+describe('Server', () => {
+
+
+  test('Should be able to handle GET request', (done) => {
+
```

Comment:
I think you're generally supposed to have [expect.assertions(#) ](https://facebook.github.io/jest/docs/en/expect.html#expectassertionsnumber) in asynchronous Jest tests.